<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="enter">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_text">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>60</y>
      <width>25</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="math3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="text">
     <string>math3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thermodynamic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>thermodynamic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="technical_rebort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>technical rebort</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="material_tochnolgy">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>material tochnolgy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ethic_and_law">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ethtic and law</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="electro_machine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>electro machine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="heat_tansfer">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>heat transfer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="human_right">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>human right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="electromagnetic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>electromagnetic feilds</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>303</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>c++</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vibration">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>vibration</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fluid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>fluid </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="circuit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>circuit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="math4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>maths 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="material_science">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>228</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>mateial science</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="intro_to_mechatronics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>128</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>intro. to mechatronics</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gpa_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>97</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>37</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>GPA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="require_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>460</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>109</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>course'require</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stress_analysis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>stress analysis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mangement">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>mangement</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="method_of_engineering_analytics">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>method of engineering analytics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="design">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>design</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="circuit2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>circuit 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="economics">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>economics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="intro_system_dynamics">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>intro system dynamics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="measurement">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>measurement</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qualities">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>qualities</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="plc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>plc</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="electric_power">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>electic power</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="micro_prossing">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>micro prossing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dynamics_and_kinamatics_of_machine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>dynamics and kinamatics of machine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cad">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CAD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="environment_science">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>environment science</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fundamental_of_control_system">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>fundamental of control system</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="micro_controller">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>micro controller</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="design_mechatronic_system">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>design mechatronic system</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cnc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CNC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_36">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_37">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_38">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_39">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_40">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_41">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_42">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_43">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_44">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_45">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_46">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_47">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>level 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>level 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>103</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>level 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>103</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>level 5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_text">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>60</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>your name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
